packages/vscode-extension/package.nls.json

```diff
@@ -59,6 +59,7 @@
     "teamstoolkit.commands.accountsLink.title": "Learn More About Accounts",
     "teamstoolkit.commands.addAppOwner.title": "Add Microsoft 365 Teams App (with Microsoft Entra App) Owners",
     "teamstoolkit.commands.azureAccountSettings.title": "Azure Portal",
+    "teamstoolkit.commands.azureAccount.signOutHelp": "Azure account Sign Out has been moved to the Accounts section on the bottom left panel. To sign out Azure, hover on your Azure account email address and click Sign Out.",
```

Comment:
Proposed: Azure account Sign Out is moved to the Accounts section on the bottom left panel. To sign out of Azure, hover on your Azure account email and click Sign Out.
Logic: Simplified it by eliminating passive voice and making it little crisper to improve readability.